recipe/meta.yaml

```diff
@@ -1,79 +1,82 @@
   version: {{ version }}
 
 source:
-  - url: https://distfiles.macports.org/py-pyqt5/PyQt5_gpl-{{ version }}.tar.gz
-    sha1: f442a876794947a24474cf85eb02abd60f00f642
-    patches:
-      - patches/0003-configure.patch
-      # remove build path from .bat
-      - patches/0004-configure-winbat.patch  # [win]
-      - patches/py2_segfault.diff
-      - patches/qt5_dll.diff
-    folder: pyqt5
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}{{ version[0] }}/{{ name }}{{ version[0] }}-{{ version }}.tar.gz
+    sha256: 2a69597e0dd11caabe75fae133feca66387819fc9bc050f547e5551bce97e5be
+    folder: pyqt
 
-  - url: https://distfiles.macports.org/py-sip/sip-{{ sip_version }}.tar.gz
-    sha1: d001eb00f6dae26a952770e805b1519f61cae77d
-    patches:
-      - patches/0001-Do-not-override-CC-CXX-LINK-in-sipgen-Makefile.patch
-      # AttributeError: module 'distutils.sysconfig' has no attribute 'get_config_h_filename'
-      - patches/sip-get_config_h_filename.patch
-    folder: sip
+  - url:  https://pypi.io/packages/source/{{ sip_name[0] }}/{{ sip_name }}/{{ sip_name.replace('-', '_') }}-{{ sip_version }}.tar.gz
+    sha256: d3e4489d7c2b0ece9d203ae66e573939f7f60d4d29e089c9f11daa17cfeaae32
+    folder: pyqt_sip
 
-  - url: https://distfiles.macports.org/py-pyqt5/PyQtWebEngine_gpl-{{ webengine_version }}.tar.gz
-    sha1: 737acf93ecf39e5e2136e094a8b0a9ffa47bd2de
-    folder: pyqtwebengine
+  - url: https://pypi.io/packages/source/{{ webengine_name[0] }}/{{ webengine_name }}/{{ webengine_name }}-{{ webengine_version }}.tar.gz
+    sha256: cedc28f54165f4b8067652145aec7f732a17eadf6736835852868cf76119cc19
+    folder: pyqt_webengine
 
-  - url: https://github.com/ocefpaf/pyqt-feedstock/releases/download/v5.12.3/PyQtChart_gpl-{{ chart_version }}.tar.gz
-    sha1: 6a876ac37d0ccb5392f67669de5e790c8f9188da
-    folder: pyqtcharts
+  - url: https://pypi.io/packages/source/{{ charts_name[0] }}/{{ charts_name }}/{{ charts_name }}-{{ charts_version }}.tar.gz
+    sha256: e2cd55a8a72cef99bc0126f3b1daa914eb5f21e20a70127b6985299f1dc50107
+    folder: pyqt_charts
 
 build:
-  number: 8
-  skip: true  # [ppc64le]
+  number: 0
 
 outputs:
-  - name: pyqt5-sip
+  - name: {{ sip_name|lower }}
     version: {{ sip_version }}
-    script: build-pyqt-sip.sh   # [not win]
-    script: bld-pyqt-sip.bat  # [win]
+    script: pyqt_sip_build.sh   # [not win]
+    script: pyqt_sip_bld.bat    # [win]
     requirements:
       build:
-        - {{ compiler('cxx') }}
         - {{ compiler('c') }}
-        - make  # [not win]
+        - {{ compiler('cxx') }}
+        - python                                 # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
       host:
         - python
+        - setuptools
+        - wheel
+        - pip
       run:
         - python
+        - sip
+        - packaging
+        - toml
     test:
       imports:
         - PyQt5.sip
 
   - name: pyqt-impl
```

Comment:
Perhaps this should be renamed to `pyqt-main`? I mean, that way it'd be easier for users and developers to know that this is the package that goes with `qt-main`.

Reply:
The previous package was named `pyqt-impl`, I don't know if we should rename them

Reply:
@isuruf, thoughts about this? I think we should rename this package to make naming consistent.

Reply:
I thought we wanted to go with upstreaming name which `pyqt = qt-main + python` and `pyqtwebengine = qt-webengine + pyqt`.

Reply:
Right, I forgot about it. I'm ok with that too.

@andfoy?

Reply:
I also forgot about that one, however, we should we do with respect to pyqt-charts? Create `pyqtcharts = pyqt + pyqt-charts`?

Reply:
Then I'd say we should go with your suggestion above.